### ISSUE TITLE
Discussion: Directory Structures

### ISSUE THREAD

**Issue by RyanZim** [fnobi/ejs-cli#9]
Perhaps I'm just missing something, but it seems a bit difficult to use ejs-cli with some directory structures.

Given the following project tree:
```
/
  src
    partials
        footer.ejs
    index.ejs
    page1.ejs
  dist
    index.html
    page1.html
  package.json
```
Where the .ejs files in `partials/` are included by the files in `src/`:

How would you render all the files in `src/` (but not the ones in `partials/`) and output them to `dist/` when calling ejs-cli from an npm script in the project root?

`ejs-cli src/*.ejs --out dist/` tries to output the files to `dist/src/*.ejs`. Right now I'm using `cd src && ejs-cli *.ejs --out ../dist/`, but it seems to me there should be a better way.

Adding to our example, suppose we add a subfolder to `src/` named `archives/`. We want to render the files in `archives/`, but not the files in `partials` or `archives/partials`. How would we accomplish that?
```
/
  src
    partials
        footer.ejs
    archives
      partials
        oldfooter.ejs
      oldpage1.ejs
    index.ejs
    page1.ejs
  dist
    archives
      oldpage.html
    index.html
    page1.html
  package.json
```

I can't think of any good ways to do it.

I like this project, but I would like to see it even better!

**Comment by RyanZim** [comment]
I decided to add a `--base-dir` option: see #10.

**Comment by RyanZim** [comment]
Fixed in fb293ee092bd70e10dfbfb23789820185a514072
